dartagnan/src/main/java/com/dat3m/dartagnan/verification/VerificationTask.java

```diff
@@ -52,6 +57,7 @@ public static VerificationTaskBuilder builder() {
     public static class VerificationTaskBuilder {
         protected WitnessGraph witness = new WitnessGraph();
         protected ConfigurationBuilder config = Configuration.builder();
+        protected ProgressModel progressModel = ProgressModel.getDefault();
```

Comment:
This is probably causing the problem with not correctly setting the FP in the CI

Reply:
No, it is because `ProgressModel` is part of the `VerificationTask` and not "just a configuration". The main function makes sure to extract the configured progress model and make it explicit in the task, but if you create the task directly (like the unit tests do), this step is skipped.